Source/missiles.cpp

```diff
@@ -387,130 +387,77 @@ void RotateBlockedMissile(Missile &missile)
 	SetMissDir(missile, dir);
 }
 
-void CheckMissileCol(Missile &missile, int mindam, int maxdam, bool shift, Point position, bool nodel)
+void CheckMissileCol(Missile &missile, int minDam, int maxDam, bool shift, Point position, bool noDel)

```

Comment:
Since you're updating these variable names anyway, I'd prefer if they were even more explicit: `minDamage`, `maxDamage`, `dontDeleteOnCollision`.